swift_scality_backend/server.py

```diff
@@ -33,6 +33,21 @@
 POLICY_STUB = object()
 
 
+class ScalityDiskFileRouter(object):
+    """
+    Replacement for Swift's DiskFileRouter object.
+    Always returns Scality's DiskFileManager implementation.
+
+    Copied from SwiftOnFile::obj/server.py
+    """
+    def __init__(self, *args, **kwargs):
+        self.manager_cls = swift_scality_backend.diskfile.DiskFileManager(
+            *args, **kwargs)
+
+    def __getitem__(self, policy):
```

Comment:
Maybe the dispatching could now be done at this level instead of lower inside our bindings. Not sure that'd be backwards compatible though?